src/ensembl/src/styles/_breakpoints.scss

```diff
@@ -1,16 +1,41 @@
-/* 
-Breakpoints suggested by Foundation (copied from _settings.scss):
+/*
+Breakpoints based on the following article:
 
-$breakpoints: (
-  small: 0,
-  medium: 640px,
-  large: 1024px,
-  xlarge: 1200px,
-  xxlarge: 1440px
-);
+https://www.freecodecamp.org/news/the-100-correct-way-to-do-css-breakpoints-88d6a5ba1862/
+
+Breakpoints are 600, 900, 1200 and 1800, based on common device types
 
 */
 
-$breakpoint-small: 'screen and (min-width: 0)';
-$breakpoint-medium: 'screen and (min-width: 900px)';
-$breakpoint-large: 'screen and (min-width: 1400px)';
+@mixin for-phone-only {
+  @media (max-width: 599px) { @content; }
+}
+
+/* big phones, tablets in portrait mode */
+@mixin for-tablet-up {
+  @media (min-width: 600px) { @content; }
+}
+@mixin for-tablet-only {
+  @media (min-width: 600px) and (max-width: 899px) { @content; }
+}
```

Comment:
One general consideration: it might be a good idea to share breakpoints with javascript (the same way we share colours). For that, we might want to have the breakpoints defined as variables that are then also exported from the SCSS file.

Reply:
Should we do this change here or in another PR?